### ISSUE TITLE
Subclassing and Inheritance doesn't mention super

### ISSUE THREAD

**Issue by codedragon** [UWPCE-PythonCert/PythonCertDevel#46]
The super class is referred to as the super class name, but not as super: ```super().methodName(args)```

**Comment by PythonCHB** [comment]
Yeah, super() is a complex topic, which will be covered in the next quarter.

But students WILL find it, so probably should at least mention it in it’s simple form here.
